Library/Formula/buku.rb

```diff
@@ -0,0 +1,18 @@
+class Buku < Formula
+  desc "Command-line bookmark manager"
+  homepage "https://github.com/jarun/Buku"
+  url "https://github.com/jarun/Buku/archive/1.7.tar.gz"
+  sha256 "0baf1594b1bf4bf47dc6b6c1e011a49e07bb6f8b6ce5f637c321f17382d9fdb2"
+
+  depends_on :python3
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    ENV["HOME"] = testpath
```

Comment:
> This should occur automatically.

Actually I didn't know that. Removed.

Quoting [formula cookbook](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#add-a-test-to-the-formula):

> The test do block automatically creates and changes to a temporary directory which is deleted after run.

Maybe worth mentioning that `$HOME` is also changed to `testpath`?